### PR TITLE
Cleaner sshd_config state

### PR DESCRIPTION
**Pull request by dwoz**

### CODE REVIEW

File: sshd_config.sls [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  {%- else %}
+    - mode: insert
+    - location: end
+  {%- endif %}
+
+sshd_config.ClientAliveCount:
+  file.line:
+    - name: {{ ssh_config }}
+    - content: "ClientAliveCountMax 20"
+  {%- if salt['file.search']('/home/dwoz/sshd_config', 'ClientAliveCountMax') %}
```
Comment (felippeb) [inline sshd_config.sls]: do we want that to be /etc/sshd/?

File: sshd_config.sls [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  {%- else %}
+    - mode: insert
+    - location: end
+  {%- endif %}
+
+sshd_config.TCPKeepAlive:
+  file.line:
+    - name: {{ ssh_config }}
+    - content: "TCPKeepAlive yes"
+  {%- if salt['file.search']('/home/dwoz/sshd_config', 'TCPKeepAlive') %}
```
Comment (felippeb) [inline sshd_config.sls]: do we want that to be /etc/sshd/?

File: sshd_config.sls [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{% set ssh_config = '/etc/ssh/sshd_config' %}
+
+sshd_config.ClientAliveInterval:
+  file.line:
+    - name: {{ ssh_config }}
+    - content: "ClientAliveInterval 60"
+  {%- if salt['file.search']('/home/dwoz/sshd_config', 'ClientAliveInterval') %}
```
Comment (felippeb) [inline sshd_config.sls]: do we want that to be /etc/sshd/?

Comment (dwoz) [reply sshd_config.sls]: Yeah, I was fixing this while you were commenting :)

File: sshd_config.sls (hunk 1 above)
Comment (bryceml) [reply sshd_config.sls]: /etc/ssh/sshd_config

**Pull request merged**